decidim-assemblies/app/serializers/decidim/assemblies/open_data_assembly_serializer.rb

```diff
@@ -0,0 +1,143 @@
+
+      # Public: Initializes the serializer with an Assembly instance.
+      def initialize(assembly)
+        @assembly = assembly
+      end
+
+      # Public: Exports a hash with the serialized data for this assembly.
+      def serialize
+        {
+          id: assembly.id,
+          slug: assembly.slug,
+          hashtag: assembly.hashtag,
+          decidim_organization_id: assembly.decidim_organization_id,
+          title: assembly.title,
+          subtitle: assembly.subtitle,
+          weight: assembly.weight,
+          short_description: assembly.short_description,
+          description: assembly.description,
+          remote_hero_image_url: Decidim::Assemblies::AssemblyPresenter.new(assembly).hero_image_url,
+          remote_banner_image_url: Decidim::Assemblies::AssemblyPresenter.new(assembly).banner_image_url,
+          promoted: assembly.promoted,
+          developer_group: assembly.developer_group,
+          meta_scope: assembly.meta_scope,
+          local_area: assembly.local_area,
+          target: assembly.target,
+          decidim_scope_id: assembly.decidim_scope_id,
+          paticipatory_scope: assembly.participatory_scope, # intentionally misspelled
+          participatory_structure: assembly.participatory_structure,
+          scopes_enabled: assembly.scopes_enabled,
+          private_space: assembly.private_space,
+          reference: assembly.reference,
+          purpose_of_action: assembly.purpose_of_action,
+          composition: assembly.composition,
+          duration: assembly.duration,
+          participatory_scope: assembly.participatory_scope,
+          included_at: assembly.included_at,
+          closing_date: assembly.closing_date,
+          created_by: assembly.created_by,
+          creation_date: assembly.creation_date,
+          closing_date_reason: assembly.closing_date_reason,
+          internal_organisation: assembly.internal_organisation,
+          is_transparent: assembly.is_transparent,
+          special_features: assembly.special_features,
+          twitter_handler: assembly.twitter_handler,
+          instagram_handler: assembly.instagram_handler,
+          facebook_handler: assembly.facebook_handler,
+          youtube_handler: assembly.youtube_handler,
+          github_handler: assembly.github_handler,
+          created_by_other: assembly.created_by_other,
+          decidim_assemblies_type_id: assembly.decidim_assemblies_type_id,
+          area: {
+            id: assembly.area.try(:id),
+            name: assembly.area.try(:name) || empty_translatable
+          },
+          scope: {
+            id: assembly.scope.try(:id),
+            name: assembly.scope.try(:name) || empty_translatable
+          },
+
+          assembly_categories: serialize_categories,
+          attachments: {
+            attachment_collections: serialize_attachment_collections,
+            files: serialize_attachments
+          },
+          announcement: assembly.announcement
```

Comment:
In the processes we export the `participatory_process_type`, yet here we do not export the assembly type. Is this intended ?

Reply:
No, it was not intended, I had just (lazily) copied from AssemblySerializer 😄 

I have added the type: adc569ef69c4e44069f35e01e93d82043b69188f